WorkflowAdminBundle/Controller/Admin/StatusController.php

```diff
@@ -48,31 +48,35 @@ public function formAction(Request $request, $statusId)
     /**
      * @param Request $request
      *
-     * @Config\Route("/status/new", name="open_orchestra_backoffice_status_new")
+     * @Config\Route("/status/new", name="open_orchestra_workflow_admin_status_new")
      * @Config\Method({"GET", "POST"})
      *
      * @return Response
      */
     public function newAction(Request $request)
     {
-        $statusClass = $this->container->getParameter('open_orchestra_model.document.status.class');
+        $statusClass = $this->getParameter('open_orchestra_model.document.status.class');
         /** @var StatusInterface $status */
         $status = new $statusClass();
         $this->denyAccessUnlessGranted(ContributionActionInterface::CREATE, $status);
```

Comment:
$this->denyAccessUnlessGranted(ContributionActionInterface::CREATE, StatusInterface::ENTITY_TYPE);